sunpy/timeseries/sources/goes.py

```diff
@@ -1,36 +1,32 @@
-# -*- coding: utf-8 -*-
```

Comment:
Given this file has unicode characters in it, we should probably keep the coding? I can't remember what the best practice on this is.

Reply:
I recall if its python3, its set to that encoding by default.

Reply:
That does seem to be the case.